Source/Renderer/UniformState.js

```diff
@@ -803,6 +802,7 @@ define([
         uniformState._inverseProjectionDirty = true;
         uniformState._inverseProjectionOITDirty = true;
         uniformState._viewProjectionDirty = true;
+        uniformState._inverseViewProjectionDirty = true;
```

Comment:
These changes in `UniformState` didn't affect the issue at hand, but they may be good changes to make.